src/test/java/software/amazon/documentdb/jdbc/sshtunnel/DocumentDbSshTunnelClientTest.java

```diff
@@ -124,8 +125,7 @@ void testInvalidSshUserAuthFail() throws Exception {
         final Exception e = Assertions.assertThrows(
                 SQLException.class,
                 () -> new DocumentDbSshTunnelClient(properties));
-        Assertions.assertEquals("java.sql.SQLException: Auth fail for methods 'publickey,gssapi-keyex,gssapi-with-mic'",
-                e.toString());
+        Assertions.assertTrue(e.toString().startsWith("java.sql.SQLException: Auth fail for methods 'publickey"));
```

Comment:
should we check based on OS? in case the auth does not fail where expected

Reply:
Yep, added.